src/dvclive/live.py

```diff
@@ -391,14 +394,20 @@ def log_image(self, name: str, val):
     def log_plot(
         self,
         name: str,
-        datapoints: List[Dict],
+        datapoints: pd.DataFrame | np.ndarray | List[Dict],
         x: str,
         y: str,
         template: Optional[str] = None,
         title: Optional[str] = None,
         x_label: Optional[str] = None,
         y_label: Optional[str] = None,
+        columns: Optional[List[str]] = None,
     ):
+        # Convert the given datapoints to List[Dict]
+        datapoints = convert_datapoints_to_list_of_dicts(
+            datapoints=datapoints, columns=columns
+        )
```

Comment:
Can we check here if it's a list first and skip in that case? That way, you can still plot a list without needing pandas and numpy installed.

Reply:
Or you can leave this and check it within that function itself. For example:

https://github.com/iterative/dvclive/blob/84069208144be711ef0feccc38064652d732eac7/src/dvclive/live.py#L372-L389

Reply:
Ok, it makes sense!  Should we raise an error if none of the supported formats `List[Dict] |pd.DataFrame | np.ndarray`  provided?

Reply:
I guess an error will already get raised below:

https://github.com/iterative/dvclive/blob/19b71b4c05c0e24f747ee4df4e331f960a4dbd97/src/dvclive/live.py#L402

However, I don't think it's a bad idea to check the types here and raise if it's not one of those. Not a blocker to me however you want to handle it.